docs/tutorial/tutorial_authentication.php

```diff
@@ -17,7 +17,7 @@
             );
     foreach ( $status as $line )
     {
-        list( $key, $value ) = each( $line );
+        $value = current( $line ); $key = key( $line );
```

Comment:
Could you please split these out over two lines? Like you have done at https://github.com/zetacomponents/Authentication/pull/3/files#diff-c21119c66b02bb326fecadc6edc9dabbR266 ?